### PR TITLE
Improvement on maven poms

### PR DESCRIPTION
**Pull request by kaicao**
- Move dependencies versions to parent/sub-parent level properties
- Remove duplicate definition of groupID in sub modules
- Fix source plugin missing groupID

### CODE REVIEW

File: regression-lib/pom.xml [hunk 1]
```diff
@@ -57,6 +56,8 @@
 			<artifactId>junit</artifactId>
```
Comment (kaicao) [inline regression-lib/pom.xml]: Should junit also have test scope instead?

**Comment by kaicao** [comment]
Looking for advices:
- There are duplicate definitions of dependencies e.g. esperio pom.xml and its submodules pom.xml, would be nice to just keep in parent the dependencies that required for all submodules by dependency management. But I don't know whether there is risk to perform such a change.
- Plugins have been defined all over the place, which might be better idea to use plugin management to keep the configuration of plugins in one/few places (e.g. parent pom.xml). But there seem to be some differences of plugin definitions here and there, again I feel there is risk to perform such a change.

**Comment by kaicao** [comment]
I’m sorry that I didn’t find the rule for creating PR to Esper and just made one here. Please let me know if I need to take a step back and follow any procedure.

**Comment by bernhardttom** [comment]
Can you sign a contributor license agreement?

**Comment by kaicao** [comment]
Sure

**Comment by bernhardttom** [comment]
The document is at http://static.espertech.com/cla/CLA.pdf

**Comment by kaicao** [comment]
Thank you @bernhardttom I have signed CLA and sent to info@espertech.com and in the meantime updated the PR to resolve the conflict.

**Pull request merged**